.ci/checker-framework.groovy

```diff
@@ -112,7 +112,8 @@ private static int checkCheckerFrameworkReport(final String profile) {
  */
 private static List<List<String>> getCheckerFrameworkErrors(final String profile) {
     final List<String> checkerFrameworkLines = new ArrayList<>()
-    final String command = "mvn -e --no-transfer-progress clean compile -P${profile}"
+    final String command = "mvn -e --no-transfer-progress clean compile"
+        + " -P${profile},no-validations"
```

Comment:
CI is failing, `+` has to be on previous line.
https://github.com/checkstyle/checkstyle/actions/runs/4097261782/jobs/7065502375#step:6:15

Reply:
Yes, we should disable this rule one day in future.

Reply:
My understanding was that this is something in groovy, not a check we turned on.
https://stackoverflow.com/a/5079914/1016482

Reply:
I rebased and fixed the code.

Reply:
thanks a lot, I just pushed a fix and got surprised that I have no PR